ui/app/selectors/selectors.js

```diff
@@ -11,6 +11,11 @@ import {
   checksumAddress,
   getAccountByAddress,
 } from '../helpers/utils/util';
+import {
```

Comment:
Review note: this is just the code from the hook, now as a selector, it doesn't appear to be modified except in name, and that it is now a selector that expects state as an argument.